ssawg_trending_scraper.py

```diff
@@ -0,0 +1,376 @@
+    tts = [tt for tt in soup.find_all("tt")]
+    return tts
+
+def get_divs(soup, identifier, url):
+    divs = [div for div in soup.find_all("div")]
+    return divs
+
+def get_ems(soup, identifier):
+    ems = [em for em in soup.find_all("em")]
+    return ems
+
+def get_scripts(soup, identifier):
+    scripts = [scripts for scripts in soup.find_all("script")]
+    return scripts
+
+# ---------------------------------
+# ESTABLISH HTML INFO FOR EACH PAGE
+# ---------------------------------
+
+class Page():
+    def __init__(self,page):
+        self.identifier = page[0:-1]
+        self.page = page
+        self.url = proper_url(self.page, base_url)
+        if self.page == "periscope_drift_reports/":
+            self.url_text = requests.get(self.url, auth=("periscope", "aca")).text
+        else:
+            self.url_text = requests.get(self.url).text
+        self.soup = BeautifulSoup(self.url_text, "lxml")
+        self.title = get_title(self.soup,self.url)
+        self.images = get_images(self.soup, self.url)
+        self.headers2 = get_headers2(self.soup, self.url)
+        self.headers3 = get_headers3(self.soup, self.url)
+        self.headers4 = get_headers4(self.soup, self.url)
+        self.tables = get_tables(self.soup,self.identifier, self.url)
+        self.paragraph = get_paragraph(self.soup, self.identifier)
+        self.anchors = get_anchors(self.soup, self.identifier)
+        self.tts = get_tt(self.soup, self.identifier)
+        self.divs = get_divs(self.soup, self.identifier, self.url)
+        self.ems = get_ems(self.soup, self.identifier)
+        self.scripts = get_scripts(self.soup, self.identifier)
+        web_block = {}
+        self.web_block = web_block[self.identifier] = {
+                                           "URL": self.url,
+                                           "HTML_URL": "<a href = " + str(self.url) + "> " + str(self.url)  + "</a>" + "<br>",
+                                           "TITLE": self.title,
+                                           "IMAGES": self.images,
+                                           "HEADERS2": self.headers2,
+                                           "HEADERS3": self.headers3,
+                                           "HEADERS4": self.headers4,
+                                           "TABLES": self.tables,
+                                           "PARAGRAPH": self.paragraph,
+                                           "ANCHORS": self.anchors,
+                                           "TTS": self.tts,
+                                           "DIVS": self.divs,
+                                           "EMS": self.ems,
+                                           "SCRIPTS": self.scripts,
+        }
+
+
+trending_blocks = {}
+for trending_page in trending_pages:
+    trending_blocks[trending_page[:-1]] = Page(trending_page)
+    
+
+# --------------------------------------------
+# CREATE CONSOLIDATED BLOCK OF NECESSARY WEB
```

Comment:
We don't need to shout in Python.  :smile:  You can just write normal capitalized sentences.